src/plugins/discover/public/locator.test.ts

```diff
@@ -58,7 +58,7 @@ describe('Discover url generator', () => {
   test('can specify specific data view', async () => {
     const { locator } = await setup();
     const { path } = await locator.getLocation({
-      indexPatternId: dataViewId,
```

Comment:
thx for cleaning this up 👍 , one note, now there is no test left for the `indexPatternId` legacy param , I think it should also be covered